commands/cloudapi-v6/image.go

```diff
@@ -159,7 +168,7 @@ func ImageCmd() *core.Command {
 		Aliases:    []string{"d"},
 		ShortDesc:  "Delete an image",
 		LongDesc:   "Use this command to delete a specified Image.\n\nRequired values to run command:\n\n* Image Id",
-		Example:    "placeholder", // TODO
+		Example:    "ionosctl image delete --image-id IMAGE_ID", // TODO
```

Comment:
should the TODO be removed? or is there anything to be added there?

Reply:
good catch, TODO should have been removed, thank you!